.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -85,9 +85,9 @@ Add any other context and details here.
 
 ### Environment
 
-- Sequelize version: XXX <!-- run `npm list sequelize` to obtain this -->
+- Sequelize version: XXX <!-- run `npm list sequelize` (or `yarn list --pattern sequelize`) to obtain this -->
 - Node.js version: XXX <!-- run `node -v` to obtain this -->
-- If TypeScript related: TypeScript version: XXX <!-- run `npm list typescript` to obtain this -->
+- If TypeScript related: TypeScript version: XXX <!-- run `npm list typescript` (or `yarn list --pattern typescript`) to obtain this -->
```

Comment:
What about `npm list typescript` / `yarn list --pattern typescript` (slash instead of or + parenthesis) to put them on equal levels